src/bindfs.1

```diff
@@ -142,6 +142,39 @@ them completely.
 See \fB\%PERMISSION \%SPECIFICATION\fP below for details.
 
 
+.SH FILE FILTERING POLICY
+
+.TP
+.B \-\-file-filter=\fIr:*.png/hiddenfile/d:dir2/...\fP, -o file-filter=...
+Takes a slash-separated list of file names which will be hidden
+in target filesystem. Files will not appear in directories, any
+access operation will fail with \fIENOENT\fP, any write operation
+(create, link, rename, ...) will fail with \fIEPERM\fP, even if
+specified file not exist. If \fI\-\-resolve\-symlinks\fP specified,
+symlinks referenced to a hidden files will also be hidden.
+
+Files can be specified as a POSIX wildcards (see \fBglob\fP(7) for
+details). Optionally, only certain file types might be matched, as
+specified with the following tokens which preceded filename and
+terminated with a colon:
+
+    \fBa\fP    any file (this is a default)
+    \fBs\fP    socket
+    \fBl\fP    symbolic link
+    \fBr\fP    regular file
+    \fBb\fP    block device
+    \fBc\fP    character device
+    \fBd\fP    directory
+    \fBp\fP    named pipe (FIFO)
+    e.g. \fIr:file/d:dir/slrbcp:notdir/...\fP
+
+If  a type tokens not specified, any file types will be matched.
+If a filename contains a colon, type token part must  be  specified
+to distinguish specifier parts, e.g. \fIa:pci:0000:00:00/...\fP,
+where \fIa\fP is an "any type" specifier and \fIpci:0000:00:00\fP is
+a file name.
```

Comment:
If we reserve `/` as the separator, we can't support filtering by paths later (without changing the flag name).
Does FUSE's option parser support reading multiple instances of the same flag? Then we wouldn't need a separator.
I'd also suggest a more verbose but more extensible and arguably clearer syntax like

    --file-filter=type=socket,name-glob=*.sock --file-filter=type=symlink,name=something

Reply:
Existing options syntax, such as --map and permission stuff tends to be concise (and I'd like it), I'll try to follow this style. Maybe better just escape forward slashes?